packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx

```diff
@@ -4,66 +4,95 @@ import { Button } from './buttons/Button';
 import { ScrollOnboardingVersion } from '../lib/featureValues';
 import { cloudinary } from '../lib/image';
 import AnalyticsContext from '../contexts/AnalyticsContext';
-import { AnalyticsEvent, TargetType } from '../lib/analytics';
+import { AnalyticsEvent } from '../lib/analytics';
+import SuperchargeIcon from '../../icons/supercharge.svg';
 
-const GaussianBlur = (): ReactElement => {
-  return (
-    <div className="absolute w-52 h-52 rounded-full opacity-32 blur-2xl bg-theme-color-cabbage" />
-  );
-};
+type ScrollOnboardingVersionMap = Partial<
+  Record<ScrollOnboardingVersion, string>
+>;
 
-const versionToContainerClassName: Record<ScrollOnboardingVersion, string> = {
-  [ScrollOnboardingVersion.V1]: 'h-60',
-  [ScrollOnboardingVersion.V2]: 'h-[36rem]',
+const versionToContainerClassName: ScrollOnboardingVersionMap = {
+  [ScrollOnboardingVersion.V1]: 'h-48 my-5',
+  [ScrollOnboardingVersion.V2]: 'h-80 m-10',
 };
-const versionToButtonClassName: Record<ScrollOnboardingVersion, string> = {
+const versionToButtonClassName: ScrollOnboardingVersionMap = {
   [ScrollOnboardingVersion.V1]: 'w-[16.25rem]',
   [ScrollOnboardingVersion.V2]: 'w-40',
 };
-const versionToButtonText: Record<ScrollOnboardingVersion, string> = {
+const versionToButtonText: ScrollOnboardingVersionMap = {
   [ScrollOnboardingVersion.V1]: 'Customize your feed',
   [ScrollOnboardingVersion.V2]: 'Start',
 };
 
+const versionToGaussianBlurClassName: ScrollOnboardingVersionMap = {
+  [ScrollOnboardingVersion.V1]: 'top-0 right-0 bottom-0 left-0 m-auto',
+  [ScrollOnboardingVersion.V2]: '',
+};
+
+const GaussianBlur = ({
+  version,
+}: {
+  version: ScrollOnboardingVersion;
+}): ReactElement => {
+  return (
+    <div
+      className={classNames(
+        'absolute w-48 h-48 rounded-full blur-2xl opacity-[0.2] bg-theme-color-cabbage',
+        versionToGaussianBlurClassName[version],
+      )}
+    />
+  );
+};
+
 interface ScrollFeedFiltersOnboardingProps {
   version: ScrollOnboardingVersion;
   onInitializeOnboarding: () => void;
 }
-export default function scrollFeedFiltersOnboarding({
+export default function ScrollFeedFiltersOnboarding({
   version,
   onInitializeOnboarding,
 }: ScrollFeedFiltersOnboardingProps): ReactElement {
   const { trackEvent } = useContext(AnalyticsContext);
   useEffect(() => {
     trackEvent({
-      event_name: AnalyticsEvent.Impression,
-      target_type: TargetType.ScrollBlock,
+      event_name: AnalyticsEvent.EligibleScrollBlock,
       target_id: version,
     });
-  }, []);
+  }, [version]);
```

Comment:
Thinking about this, we might need to add a state to ensure it only triggers once, not on re-render.
There are some examples of tracking where we do that.

Reply:
As long as component does not remount this won't trigger again. This will trigger if user goes to other page and then back to the homepage (because of the unmount of course), but I think in that case it should trigger since users encounters the scroll block again.

If we want to trigger it only once per session let me know.

Reply:
also `version` currently does not change until next full page reload so this is just a case if in future we have a dynamic experiments added so it actually responds to the version change, more of a future proofing.

Reply:
Ah yep for this type of component it's actually alright, never mind this comment.